File: remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        p = 0
        n = len(nums)
        for i in range(1, n):
            if nums[i]!=nums[i-1]:
                nums[p]=nums[i-1]
                p+=1
        nums[p]=nums[n-1]
        p+=1
        return p

File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.java
class Solution {
    // void swap(int[] arr, int i, int j) {
    //     int temp = arr[i];
    //     arr[i] = arr[j];
    //     arr[j] = temp;
    // }

    public int removeDuplicates(int[] nums) {
        int l = 0;
        for(int i: nums){
            if(nums[l]!=i) {
                nums[l+1]=i;
                l++;
            }
        }
        return l+1;
    }
}